### ISSUE TITLE
Not enough arguments for EV::timer with perl 5.36

### ISSUE THREAD

**Issue by perlpunk** [powerman/perl-Test-Mock-Time#9]
I'm getting an error under perl 5.36 on openSUSE Tumbleweed.

```
use EV;
use Test::Mock::Time;
```
```
Not enough arguments for EV::timer at /usr/lib/perl5/vendor_perl/5.36.0/Test/Mock/Time.pm line 293, near "@_)"
BEGIN not safe after errors--compilation aborted at /usr/lib/perl5/vendor_perl/5.36.0/Test/Mock/Time.pm line 304.
Compilation failed in require at repro.pl line 2.
BEGIN failed--compilation aborted at repro.pl line 2.
```

**Comment by powerman** [comment]
It's a bit weird story… When I've opened this project I found uncommited fix for this issue, probably created few years ago. No idea why it wasn't commited, all tests pass so I've just released v0.2.0 with this fix.

**Comment by perlpunk** [comment]
Thanks!
With [this fix](https://github.com/powerman/perl-Test-Mock-Time/commit/3ecb90c8106485dd4bea2751a228dc25fa4a585e), however, which I also tried locally before creating this issue, the script just hangs.
Unfortunately it's a bit difficult to create a small reproducer.
We have a test that is testing a Mojolicious app, and once we add `use Test::Mock::Time` it hangs now.
We only stumbled upon it because a team member had EV installed for something else; and Mojolicious loads it if it is installed; we are not even actively using it.
This is the test: https://github.com/os-autoinst/openQA/blob/master/t/api/13-influxdb.t

We are now using Test::MockModule instead, and the colleague also deinstalled EV.
Maybe it can be reproduced with a very simple Mojolicious dummy app, but currently we don't have any additional time that we can spend on this.
But I wanted to leave the info here, in case someone else has the same problem.
